### PR TITLE
Feature/router

### PR DESCRIPTION
**Pull request by Gabby-Recny**
### What (if any) features are you implementing?
- Added Router to application
### What (if anything) did you refactor?
- Removed displayMain function and replaced with Links
### Were there any issues that arose?
- Error message pops up on API Call to individual movies
### Is there anything that you need from your teammate?
- Keep being dope!
### Refactor checklist
- [ ] Check variable and parameter names: are they descriptive and unique?
- [ ] Is the code DRY? SRP?
- [ ] Is it easy for others to read your code? (explanatory comments needed?)
- [ ] Is the syntax consistent? (e.g. use of semicolons, blank lines)
- [ ] Are commit messages formatted correctly?

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -4,7 +4,9 @@ import './App.scss';
 import Header from './Components/Header/Header.js';
 import MovieContainer from './Components/MovieContainer/MovieContainer';
 import MovieDetails from './Components/MovieDetails/MovieDetails';
+import ErrorPage from './Components/ErrorPage/ErrorPage';
 import { getAllMovies, getSingleMovie } from './api-calls';
+import { Route, Switch } from 'react-router-dom';
```
Comment (anewb87) [inline src/App.js:9]: Switch it uuuuup!

File: src/App.js [hunk 2]
```diff
@@ -24,22 +25,34 @@ class App extends React.Component {
       .catch(error => this.setState({ error: error }))
   }
 
-  handleClick = (id) => {
+  chooseMovie = (id) => {
     getSingleMovie(id)
-      .then(cleanedData => this.setState({ selectedMovie: cleanedData, isSelected: true }))
```
Comment (anewb87) [inline src/App.js:29]: Cleaning up data that's no longer needed.

**Review by jasmyn2244 — APPROVED** [review]
great job!

File: src/App.js [hunk 3]
```diff
@@ -24,22 +25,34 @@ class App extends React.Component {
...
-        {this.state.isSelected && <MovieDetails selectedMovie={this.state.selectedMovie} displayMain={this.displayMain} />}
-        {!this.state.isSelected && <MovieContainer movies={this.state.movies} chooseMovie={this.handleClick} />}
-        {this.state.error && <h2>Sorry you got an error. Please try again.</h2>}
+          {this.state.error && <h2>Sorry you got an error. Please try again.</h2>}
+        <Switch>
+          {/* Console Logs are showing up twice in clicking on a movie 
+          If it wasn't getting logged twice, we would only get the error page.
+          May need an async function to make sure we're seeing a loader while API calls is loading.
+          */}
+          <Route exact path="/:movie" render={ ( { match } ) => {
```
Comment (anewb87) [inline src/App.js:43]: This is super cool! Crazy! Router for the win!

File: src/Components/ErrorPage/ErrorPage.js [hunk 4]
```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import './ErrorPage.scss';
+import { Route, Link } from 'react-router-dom';
+
+
+const ErrorPage = ( { backHomeBtn }) => {
```
Comment (anewb87) [inline src/Components/ErrorPage/ErrorPage.js:6]: Adding an error page.

**Review by anewb87 — APPROVED** [review]
Getting on the Router train!

**Pull request merged**